MetaMask/metamask-extension
# UX Multichain: added extra margin for unlocked state in full view in header

The header in some screens was overlapping with the content. This PR is to fix the overlapping between content and header by adding a padding of 16px for full view screen in unlocked state.

* Fixes MetaMask/MetaMask-planning#850 

## Screenshots/Screencaps

### Before
![Screenshot 2023-06-21 at 2 25 24 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/d22c2472-124c-4997-b406-f8ac2c4f7975)


### After

![Screenshot 2023-06-21 at 2 19 48 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/3e7a38df-382f-4fa4-9bfe-ee6f4a173470)

## Manual Testing Steps

- Go to the confirmation screen or any screen in full view
- Check the gap between the content and the header

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
